classification/train.py

```diff
@@ -282,36 +281,34 @@ def main():
     """
     torch.multiprocessing.set_start_method('spawn')
     print("Device is: ",device)
-    CONFIG = get_config()
     # Needed to redefine wandb_run as a global variable
     # pylint: disable=global-statement
     global wandb_run
-    wandb_run = init_wandb(CONFIG)
-    set_seed(CONFIG.seed)
+    wandb_run = init_wandb()
+    set_seed(cfg.seed)
 
     # Load in dataset
     print("Loading Dataset")
     # pylint: disable=unused-variable
     # for future can use torchvision.transforms.RandomApply here
     transforms = torch.nn.Sequential(SyntheticNoise("white", 0.05))
-    train_dataset, val_dataset = get_datasets(transforms=transforms, CONFIG=CONFIG)
-    train_dataloader, val_dataloader = load_datasets(train_dataset, val_dataset, CONFIG)
+    train_dataset, val_dataset = get_datasets(transforms=transforms)
+    train_dataloader, val_dataloader = load_datasets(train_dataset, val_dataset)
 
     print("Loading Model...")
     model_for_run = TimmModel(num_classes=train_dataset.num_classes, 
-                                model_name=CONFIG.model, 
-                                CONFIG=CONFIG).to(device)
+                              model_name=cfg.model).to(device)
     model_for_run.create_loss_fn(train_dataset)
-    if CONFIG.model_checkpoint is not None:
-        model_for_run.load_state_dict(torch.load(CONFIG.model_checkpoint))
-    optimizer = Adam(model_for_run.parameters(), lr=CONFIG.lr)
+    if cfg.model_checkpoint != "":
+        model_for_run.load_state_dict(torch.load(cfg.model_checkpoint))
+    optimizer = Adam(model_for_run.parameters(), lr=cfg.learning_rate)
     scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optimizer, eta_min=1e-5, T_max=10)
     print("Model / Optimizer Loading Successful :P")
     
     print("Training")
     best_valid_map = 0
-    early_stopper = EarlyStopper(patience=CONFIG.patience, min_delta=CONFIG.min_delta)
-    for epoch in range(CONFIG.epochs):
+    early_stopper = EarlyStopper(patience=cfg.patience, min_delta=cfg.min_valid_map_delta)
+    for epoch in range(cfg.epochs):
         print("Epoch " + str(epoch))
 
```

Comment:
add a comment about why both of these are best_valid_map

Reply:
I think that's kinda outside this pr, we should put that in code review hotfixes